scripts/dotnet-cli-build/Utils/DotNetCli.cs

```diff
@@ -37,11 +37,11 @@ private static string GetStage0Path()
             {
                 return Path.Combine(Directory.GetCurrentDirectory(), ".dotnet_stage0",
                     PlatformServices.Default.Runtime.OperatingSystemPlatform.ToString(),
-                    PlatformServices.Default.Runtime.RuntimeArchitecture, "cli", "bin");
```

Comment:
I don't have a reason :). Feel free to unify if everything else works smoothly with it :D. Unification FTW